### PR TITLE
Fixed the log

### PR DESCRIPTION
**Pull request by AdriVanHoudt**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1 @@
-console.log('Vladimir is the coolest (and the most modest) Node.js dev in the world');
+console.log('Vladimir is the "coolest" (and the "most modest") Node.js dev in the world');
```
Comment (vdeturckheim) [inline index.js]: Can you remove all your changes and add
`console.log('I am Adri and I approve this message')` ?

**Review by vdeturckheim — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (AdriVanHoudt) [reply index.js]: fixed

**Comment by AdriVanHoudt** [comment]
oh no you didn't :O

**Comment by vdeturckheim** [comment]
I will close until to accept the style guide of this project.

**Comment by AdriVanHoudt** [comment]
Hey you have no contributing.md!

**Comment by vdeturckheim** [comment]
You're right, I just fixed that

**Pull request closed without merging**